pkg/aws/helpers.go

```diff
@@ -520,6 +521,7 @@ func GetOperatorRolePolicyPrefixFromCluster(cluster *cmv1.Cluster, awsClient Cli
 		}
 		attachedPoliciesDetail := FindAllAttachedPolicyDetails(policiesDetails)
 		for _, attachedPolicyDetail := range attachedPoliciesDetail {
+			policyNames = append(policyNames, attachedPolicyDetail.PolicyName)
 			index := strings.LastIndex(attachedPolicyDetail.PolicyName, "-openshift")
```

Comment:
We still look for "-openshift" suffix for policies over here

Reply:
can we remove it ?

Reply:
We can, but it is not necessary. This was to illustrate to @oriAdler that this was still being done for the sake of precedence.